.dagger/consts/versions.go

```diff
@@ -9,8 +9,9 @@ const (
 )
 
 const (
-	GolangVersion = distconsts.GolangVersion
-	GolangImage   = distconsts.GolangImage
+	GolangVersion   = distconsts.GolangVersion
+	GolangImage     = distconsts.GolangImage
+	TypescriptImage = "node:22.11.0-alpine"
```

Comment:
this really shouldn't be re-declared here, its actual source of truth is currently in `sdks/typescript/runtime/main.go`. is it inadvisable to link that here? should these be functions in `typescript-sdk-dev`? I think some of the other module sdks do it that way.

Reply:
Interesting question! The source of trust should indeed be from `runtime/main.go` but, because you only install TSX prior to calling the TS runtime, you need a default base image at some point.
I think it shouldn't be defined here, because it creates some assumptions that the TS runtime image static while it's not the case.
Maybe just inline it in `build/sdk.go` as a base image to install utility tools to the runtime.

Reply:
> I think it shouldn't be defined here, because it creates some assumptions that the TS runtime image static while it's not the case.

yeah we're saying the same thing, but `build/sdk.go` is only marginally better... is there something wrong with putting it in typescript-sdk-dev and having both runtime/main.go and build/sdk.go import that? or importing `sdks/typescript/runtime/` here in `build/sdk.go`? (fwiw the second one _feels_ wrong to me)